### ISSUE TITLE
"Some of your messages have not been sent" but no unsent messages to be found

### ISSUE THREAD

**Issue by babolivier** [vector-im/element-web#19675]
When I opened Element this morning I saw this when displaying `#synapse-dev`:

![image](https://user-images.githubusercontent.com/5547783/141081792-48f67680-8fe6-41be-b922-16c737dd8e2f.png)

But a) the dialog doesn't show me which messages have failed to send (my latest message in the room was yesterday morning), and b) scrolling up on the timeline doesn't show any message with a red circle in front of them (which iirc is what should show when a message fails to send?).

**Comment by babolivier** [comment]
Have sent logs

**Comment by babolivier** [comment]
Sorry I realise I forgot to mention the version I was running :man_facepalming: it's whatever was running on develop.element.io, currently it's

Element version: 28f00498a2b2-react-71d894664113-js-b07d44a6c057
Olm version: 3.2.3

But I'm not sure if it's the same now as it was then

**Comment by dbkr** [comment]
This will probably be a reaction or some similar event that isn't rendered as its own tile that's failed to send.

**Comment by novocaine** [comment]
I think this is most likely a duplicate of https://github.com/vector-im/element-web/issues/18403

**Comment by babolivier** [comment]
> I think this is most likely a duplicate of #18403

I'm not sure it is, actually. It's happened just now in HQ, in which I interact little enough to be able to easily remember what I've sent. With a new account, I sent exactly 3 messages (4 if counting edits) and no other event (so no reaction, etc) and all are showing up as sent in the timeline. But when I opened Element Web a few minutes ago, it showed me the "Some of your messages have not been sent". I've also checked with another account and all of these messages show as sent.

I've sent a rageshake about this.
